messages/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "buyer-organizations-supplier.admin.navigation.label-group": "Organizations",
```

Comment:
I'd suggest referencing "organizations" somehow in the label's name :)
It could make more sense as we add new navigation labels in the future.